scania-digital-design-system/sdds
# fix(banner): moving theme rules to component folder

**Describe pull-request** 
Moving theming rules to the component folder 

**Solving issue** 
Fixes: [AB#2213](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2213)

**How to test** 
1. Open banner Storybook of this branch 
3. Open banner Storybook of production 
4. 3. Compare styling- there should not be any difference between those two
